torchmultimodal/modules/layers/transformer.py

```diff
@@ -115,10 +121,10 @@ def __init__(
 
     def forward(
         self,
-        hidden_states,
-        attention_mask=None,
-        head_mask=None,
-    ):
+        hidden_states: Tensor,
+        attention_mask: Tensor = None,
```

Comment:
should be optional no? why isnt it getting flagged tho?

Reply:
There is a [flag](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-strict-optional) for this that we have set right now. Turning it off triggers a ton of errors so I'm gonna leave it as is for now. We can handle that in a follow-up if we want

Reply:
oh no there's more? :(